modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -17,11 +17,17 @@ You can also use the Agent-based Installer to generate or accept Zero Touch Prov
 * You read the documentation on xref:../../installing/installing-preparing.html#installing-preparing-selecting-cluster-type[Selecting a cluster installation method and preparing it for users].
 * You have read the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[Preparing to install with the Agent-based Installer] documentation.
 * You downloaded the xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Agent-Based Installer] and the command-line interface (CLI) from Red Hat’s Hybrid Cloud Console.
-* For a disconnected environment, you created a container image registry, such as Red Hat Quay. See xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-introduction_installing-mirroring-creating-registry[Mirror registry for Red Hat OpenShift introduction].
 * You have logged into the {product-title} with administrator privileges. 
 
 .Procedure
 
+. For a disconnected environment, you mirrored the {product-mirror-registry} to your local container image registry. See xref:../../installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the {product-title} image repository].
++
+[IMPORTANT]
+====
+Check that your `openshift-install` binary version relates to your local image container registry and not a shared registry such as {quay}.
+====
+
```

Comment:
A line saying how the output of `openshift-install version` should look like to confirm it comes from the local registry would be helpful for users. Wdyt?

Reply:
Updated but Hi @bfournie and @rwsu, maybe you could provided better output?

Reply:
Yes, perhaps the change I suggested in line 40 above.